notebooks/week_3/nsyns_per_conn.ipynb

```diff
@@ -274,10 +274,10 @@
    "metadata": {},
    "source": [
     "### Exercise #1\n",
-    "Calculate the average number of synapses per connection in the four classes of connections (EE, EI, IE, II). Put the answer in a list called _ans_1_. Respect the order (EE, EI, IE, II).\n",
+    "Calculate the average number of synapses per connection in the four classes of connections (EE, EI, IE, II). Put the answers in a list called _ans_1_ in the order (EE, EI, IE, II).\n",
     "\n",
     "### Exercise #2\n",
-    "Calculate the distribution of number of synapses from SP_PVBC to SP_PC. Put the answer in a list named _ans_2_ with mean and std"
+    "Calculate the distribution of number of synapses from SP_PVBC to SP_PC. Provide the (mean, std) as a 2-tuple for answer _ans_2_."
```

Comment:
> 2-tuple
?

Reply:
`Provide a list for _ans_2_, with the first element being the mean value and the second element being the standard deviation of the number of synapses from SP_PVBC to SP_PC.`

Better?